hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/webapp/FederationInterceptorREST.java

```diff
@@ -175,8 +175,7 @@ private SubClusterId getRandomActiveSubCluster(
     }
     List<SubClusterId> list = new ArrayList<>(activeSubclusters.keySet());
 
-    FederationPolicyUtils.validateSubClusterAvailability(
-        list, blackListSubClusters);
+    FederationPolicyUtils.validateSubClusterAvailability(list, blackListSubClusters);
```

Comment:
We could just do:
```
FederationPolicyUtils.validateSubClusterAvailability(activeSubclusters.keySet(), blackListSubClusters);
```

Reply:
Thanks for your suggestion, I will modify the code.